src/map.cairo

```diff
@@ -0,0 +1,32 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.math import assert_nn
+
+using Pair = (eth_address : felt, starknet_address : felt)
+
+@storage_var
+func storage(addresses: Pair) -> (exists: felt):
```

Comment:
You don't need an additional structure here. `(eth_address, starknet_address) -> exists` will work